pkg/namestore/namestore.go

```diff
@@ -96,8 +95,8 @@ func (x *nameStore) Rename(oldName, id, newName string) error {
 	if oldName == "" || newName == "" {
 		return nil
 	}
-	if err := identifiers.Validate(newName); err != nil {
-		return fmt.Errorf("invalid name %q: %w", oldName, err)
+	if err := identifiers.ValidateDockerCompat(newName); err != nil {
+		return fmt.Errorf("invalid new name: %w", err)
```

Comment:
```suggestion
		return fmt.Errorf("invalid name %q: %w", newName, err)
```

Reply:
itention was not to repeat the identifier twice as the err also prints it, doesnt harm to be verbose, updated it.